AI_interface.py

```diff
@@ -178,8 +178,22 @@ def train_model(self, starting_train_step=0):
                     global_agent.tft_save_model(train_step)
 
     def collect_dummy_data(self):
-        dataWorker = DataWorker()
-        dataWorker.collect_dummy_data()
+        env = parallel_env()
+        while True:
+            _ = env.reset()
+            terminated = {player_id: False for player_id in env.possible_agents}
+            t = time.time_ns()
+            while not all(terminated.values()):
+                # agent policy that uses the observation and info
+                action = np.random.randint(low=0, high=[10, 5, 9, 10, 7, 4, 7, 4], size=[8, 8])
```

Comment:
This array for high is going to change when Lobo's changes get pushed.. you may have to go back to the config and see what changed when that happens.